jenkins/add-docker-build-job.sh

```diff
@@ -40,6 +42,8 @@ job_short_name="basic-docker-build"
 job_display_name="Basic Docker Build"
 job_description="A basic pipeline that builds a Docker container. The job expects a Dockerfile at the root of the git repository"
 repository="${USER}/myfirstapp"
+scm_poll_schedule=""
```

Comment:
I'm fine always including this in fact. Why wouldn't you want polling? Is there some reason a user would *only* want to setup a webhook on their own and not use polling?

If you always include it, you could just add the xml below to the job xml file